script/bootstrap

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
are we renaming scripts back to script? I think `scripts` was more consistent w/ our directly pluralization